### PR TITLE
[healpix]Add new port.

### PR DESCRIPTION
**Pull request by JackBoosY**
Only support Linux. Needs autoconf.

Related: #8795.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

**Review by dan-shaw — APPROVED** [review]

File: ports/healpix/CONTROL [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Source: healpix
+Version: 1.12.10
+Homepage: http://healpix.sourceforge.net/
+Description: HEALPix is an acronym for Hierarchical Equal Area isoLatitude Pixelation of a sphere.
+Build-Depends: cfitsio
```
Comment (strega-nil) [inline ports/healpix/CONTROL]: Can you add a `Supports` line here?

**Comment by strega-nil** [comment]
Cool, looks good to me :)

**Pull request merged**